langcodes/__init__.py

```diff
@@ -98,32 +98,92 @@ def get(tag: str, normalize=True) -> 'LanguageData':
         If normalize=True, non-standard or overlong tags will be replaced as
         they're interpreted. This is recommended.
 
+        Here are several examples of language codes, which are also test cases.
+        Most language codes are straightforward, but these examples will get
+        pretty obscure toward the end.
+
         >>> LanguageData.get('en-US')
         LanguageData(language='en', region='US')
 
-        >>> LanguageData.get('sh-QU')        # transform deprecated tags
-        LanguageData(language='sr', macrolanguage='sh', script='Latn', region='EU')
+        >>> LanguageData.get('zh-Hant')
+        LanguageData(language='zh', script='Hant')
+
+        >>> LanguageData.get('und')
+        LanguageData()
+
+        The non-code 'root' is sometimes used to represent the lack of any
+        language information, similar to 'und'.
```

Comment:
I noticed that pycld2 uses "un" for this, but I don't know if anything else does.